.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python: "3.7"
+cache: pip
+
+before_install:
+  - pip install flake8
+
+script:
+  - flake8 -v .
```

Comment:
I vote to turn off `E501 - line too long` and the changes below that are motivated by that check.

Even Linus Torvalds [has commented](https://lkml.org/lkml/2020/5/29/1038) saying that longer lines are useful and that excessive line breaks caused by 80-character length limit are bad. The fact that monitors have grown in the past few decades means we don't have to work like people had to in '70s or even earlier.

Reply:
#28 is the new version of this PR with E501 turned off